packages/selinux-policies/build.yaml

```diff
@@ -0,0 +1,22 @@
+requires:
+- name: "base"
+  category: "distro"
+  version: ">=0"
+
+steps:
+{{ if .Values.distribution }}
```

Comment:
yes it should be possible to apply custom policies. Something similar to what is documented [here](https://debian-handbook.info/browse/en-US/stable/sect.selinux.html)  for the `Adapting the Rules` section. I can give it a try to see if I can adapt or provide some basic module even if it is just to meant to be an example.

Reply:
@davidcassany thx :+1: even a link to that doc would be enough for the moment

Reply:
All right, I managed to include a custom dummy cOS SELinux Targeted policy module. Now it is possible to boot cOS with enforced SELinux. The cOS module is the outcome of booting and upgrading on permissive mode without it, then I parsed audit logs with `audit2allow` tool which translates SELinux denials into some rules to avoid each specific denial. Obviously this is a bad approach, since it does not add any sort of specific hardening, it just relaxes the defaults probably in some not accurate way. I did not analyze if the result makes sense at all. I just checked I could boot with SELinux on `enforcing` mode.